packages/ecommerce-app-base/src/AppConfig/AppConfig.tsx

```diff
@@ -258,9 +259,9 @@ export default class AppConfig extends React.Component<Props, State> {
             </Form>
           </>
         )}
-        {isInOrchestrationEAP && (
+        {isInOrchestrationEAP && !hideOrchestrationEapNote && (
           <OrchestrationEapNote
-            hideOrchestrationEapNote={hideOrchestrationEapNote}
+            setHideOrchestrationEapNote={this.setHideOrchestrationEapNote.bind(this)}
```

Comment:
```suggestion
            onClose={this.setHideOrchestrationEapNote(true)}
```

And then just call it directly in the component or? i.e. 

` <Note className={styles.eapNote} withCloseButton={true} onClose={onClose} title={title}>`

Reply:
Let me try but not sure this will work. 🤔